sphere_grid_reader.py

```diff
@@ -0,0 +1,32 @@
+# Libraries and Core Files
+import memory.main
+from memory.sphere_grid import SphereNodeType, sphere_grid
+
+# Wait for memory to finish loading
+while not memory.main.start():
```

Comment:
It's probably worth wrapping most of this stuff in a function. The act of importing this file would do bad things right now. Lol.

Reply:
I added a comment clarifying that this file is not meant to be integrated into the TAS, only serve as an example on how to use the API.